lib/backend/compat/compat.go

```diff
@@ -219,6 +286,15 @@ func (c *ModelAdaptor) getNode(nk model.NodeKey) (*model.KVPair, error) {
 	return &model.KVPair{Key: nk, Value: &nv}, nil
 }
 
+// validateBlockValue validates the AllocationBlock fields (specifically Affinity) to
+// make sure the deprecated HostAffinity field is not used.
+func validateBlockValue(kvp *model.KVPair) error {
+	if kvp.Value.(*model.AllocationBlock).HostAffinity != nil {
+		return goerrors.New("HostAffinity field is deprecated, use Affinity")
```

Comment:
Tweak the wording of this error.

"_AllocationBlock_.HostAffinity is deprecated, _please use_ Affinity _instead_."